frontend/src/components/table/Table.js

```diff
@@ -90,6 +92,25 @@ class Table extends Component {
       page,
     } = this.props;
     const { selected, rows } = this.state;
+
+    /**
+     * Selection by default of first row if nothing selected
+     */
+    if (
+      (_.isEmpty(defaultSelected) || _.isEmpty(selected)) &&
+      !_.isEmpty(rows)
```

Comment:
if `rows` is an array (and same for `selected`) why can't we just check the `length` property ?

Reply:
agree. data shall be correct and valid from the beginning.